packages/trader/src/AppV2/Components/BottomNav/BottomNav.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import {
+    LegacyMarketBasketIndicesIcon,
+    LegacyMenuHamburger1pxIcon,
+    StandaloneChartCandlestickRegularIcon,
+    StandaloneClockThreeRegularIcon,
+} from '@deriv/quill-icons';
+import BottomNavItem from './BottomNavItem';
+import { Badge } from '@deriv-com/quill-ui';
+
+type BottomNavProps = {
+    className?: string;
+    children: React.ReactNode[];
+};
+
+const BottomNavItems = [
+    {
+        icon: <StandaloneChartCandlestickRegularIcon iconSize='sm' />,
+        label: <Localize i18n_default_text='Trade' />,
+    },
+    {
+        icon: <LegacyMarketBasketIndicesIcon iconSize='sm' />,
```

Comment:
When I click on the icon, it doesn't change the color. Dame for 'menu'. Should it change or not? Because half of the icons do it, half - not

Reply:
yeah currently in discussion with quill-icons team to allow fill for Legacy icons. They allow fill/color-change for standalone and some other icons but not legacy icons